fastlane/Fastfile

```diff
@@ -854,36 +854,75 @@ desc 'Show current frameworks size'
 lane :show_frameworks_sizes do |options|
   next unless is_check_required(sources: sources_matrix[:size], force_check: @force_check)
 
-  options[:sizes] ||= frameworks_sizes
+  ['metrics/'].each { |dir| FileUtils.remove_dir(dir, force: true) }
+
+  sh("git clone #{metrics_git} #{File.dirname(sdk_size_path)}")
+  sdk_size_benchmarks = JSON.parse(File.read(sdk_size_path))
+  expected_sizes = sdk_size_benchmarks['develop']
```

Comment:
How is this getting the SDK size from develop? Is it returning the current badge value from the README.md?

Reply:
From here https://github.com/GetStream/apple-internal-metrics/blob/main/stream-chat-swift-size.json